### DISCUSSION TITLE
Right mouse drag to absolute scroll

### DISCUSSION THREAD

**Discussion by Neighboryyy** [ppy/osu#25181] [Ideas] [answered]
Could this be changed to work the way it does in stable? As in it triggers _everywhere_ and not just a specified portion of the screen.

**Comment by peppy** [comment, marked as answer]
Please  go through the first run wizard. There's a setting for this.

https://github.com/ppy/osu/discussions/24757 etc.

**Reply by Neighboryyy** [reply]
If you are referring to this setting I already had it enabled. What I meant was the setting triggering everywhere in song select and not just in an area. I'm using 0,8 ui scaling so maybe that could be tied to it.
![image](https://github.com/ppy/osu/assets/148543247/a73cdf22-874b-460f-9392-65a82ccb7922)

Second picture shows roughly the line where it stops working.
![image](https://github.com/ppy/osu/assets/148543247/62460e19-67ca-49e6-9004-6d358f2fb03f)

**Reply by peppy** [reply]
I believe there is good reason to have this limited, which has been discussed in other issue threads.  We could reassess after the song select redesign, or you could just click closer to the carousel?

**Reply by peppy** [reply]
I can see that it's pretty annoying at lower UI scale. I'll see what we can do.

**Reply by peppy** [reply]
I've made a proposal at #25182 which should fix your issues.
